solutions/src/router.js

```diff
@@ -0,0 +1,71 @@
+const http = require('http');
+const users = require('./static.js');
+const fs = require('fs');
+const pg = require('pg');
+const getData = require('./queries/getData.js');
+const postData = require('./queries/postData.js');
+
+const router = (request, response) => {
+  const endpoint = request.url.split('/')[1];
+
+  if (endpoint === '') {
+    response.writeHead(200, { "Content-Type": "text/html" });
```

Comment:
1. Be consistent with how you're setting your headers. Here you use an object, in other places you set the string directly.
2. I would move this `writeHead` call into the callback. Seems odd to write it here and then overwrite it later.